ecs-init/config/common.go

```diff
@@ -32,15 +32,18 @@ const (
 
 	UnixSocketPrefix = "unix://"
 
-	//DefaultRegionName is the name of the region to fall back to if no entry for the region name is found in the
-	//S3BucketMap.
+	// AgentFileName is the filename, including version number, of the agent to be downloaded.
+	AgentFilename = "ecs-agent-v1.14.1.tar"
+
+	// DefaultRegionName is the name of the region to fall back to if no entry for the region name is found in the
+	// S3BucketMap.
 	DefaultRegionName = "default"
 )
 
-//s3BucketMap provides a mapping of region names to specific URI's for the region.
+// s3BucketMap provides a mapping of region names to specific URI's for the region.
 var s3BucketMap = map[string]string{
```

Comment:
Can you rename `s3BucketMap` to `regionToS3BucketURL`? It's good to avoid the type of the variable when naming it. Also, makes it more meaningful